controller/Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package]
 name = "epic_wallet_controller"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
```

Comment:
Change this author to Grin and Epic Developers

Reply:
We are keeping the authors just to Epic, such as is since we forked the repo